modules/gapi/src/backends/ie/giebackend.cpp

```diff
@@ -336,6 +363,17 @@ struct IEUnit {
         } else {
             cv::util::throw_error(std::logic_error("Unsupported ParamDesc::Kind"));
         }
+
+        using namespace cv::gapi::wip::onevpl;
+        if (params.vpl_preproc_device.has_value() && params.vpl_preproc_ctx.has_value()) {
+            using namespace cv::gapi::wip;
+            GAPI_LOG_INFO(nullptr, "VPP preproc creation requested");
+            preproc_engine_impl =
+                IPreprocEngine::create_preproc_engine<onevpl::VPPPreprocDispatcher>(
```

Comment:
Is preproc enabled with the received context and device parameters?
Can I use context and device for something else? Can it bring dual behavior in the future? (When I set device and context and activate preproc by chance or it is impossible case)

Reply:
to enable preprocessing we should call  `face_net.cfgPreprocessingParams(accel_device.value(), accel_ctx.value());` in explicit way
Please see  https://github.com/opencv/opencv/pull/21688/files#diff-b4563bcdbe86eb5c8f1f2cce227809b02980777b25f5f4be2bb717826bedbebbR398